docs/reference/cluster_manifest.md

```diff
@@ -122,6 +122,19 @@ These parameters are grouped directly under  the `spec` key in the manifest.
   A map of key value pairs that gets attached as [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
   to each pod created for the database.
 
+* **serviceAnnotations**
+  A map of key value pairs that gets attached as [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+  to the services created for the database cluster. If
+  `enableMasterLoadBalancer` parameter (or the operator config
```

Comment:
I think this part about service annotations when enabling load balancers better fits into [this](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#load-balancers-and-allowed-ip-ranges) section in the admin docs. Still, you might give a hint and link to this paragraph.